### PR TITLE
979 use secrets for setup test users

### PR DESCRIPTION
**Pull request by jamesobrooks**

### CODE REVIEW

File: docs/operations/production-like-data-sets.md [hunk 1]
```diff
@@ -15,5 +15,5 @@ In order to do this, you need to have an empty DDB table in the environment to w
 4. After the blue/green migration completes, you will need to recreate the test users for that environment as they are not included in the production data.
 
     ```bash
-    node shared/admin-tools/users/setup-test-users.js
+    node shared/admin-tools/users/setup-test-users.sh
```
Comment (rachelschneiderman) [inline docs/operations/production-like-data-sets.md]: Same here :)

File: docs/environments.md [hunk 2]
```diff
@@ -165,7 +165,7 @@ EF-CMS currently has both the concept of a deployment at a domain as well as a n
     node shared/admin-tools/user/setup-admin.js
     ```
     ```bash
-    node shared/admin-tools/user/setup-test-users.js
+    node shared/admin-tools/user/setup-test-users.sh
```
Comment (rachelschneiderman) [inline docs/environments.md]: Same here, would you mind just updating this example please?

File: docs/additional-resources/troubleshooting.md [hunk 3]
```diff
@@ -198,7 +198,7 @@ Problem:
 Solution: 
 - This solution should only be used on non-prod environments. Clear the dynamo tables and recreate the users and judges.
     ```bash
-    node shared/admin-tools/user/setup-test-users.js
+    node shared/admin-tools/user/setup-test-users.sh
```
Comment (rachelschneiderman) [inline docs/additional-resources/troubleshooting.md]: Nit: I think 'node' should change to './'? 😄

**Review by rachelschneiderman — APPROVED** [review]

File: docs/additional-resources/troubleshooting.md (hunk 3 above)
Comment (jamesobrooks) [reply docs/additional-resources/troubleshooting.md]: Good point. Thanks!

**Pull request merged**